deploy/helm/lunettes/templates/grafana/grafana-cm.yaml

```diff
@@ -671,7 +672,7 @@ data:
                       {
                         "targetBlank": true,
                         "title": "yaml",
-                        "url": "http://lunettes.lunettes.svc.cluster.local:8080/api/v1/debugpodyaml?uid=${__data.fields[\"PodUID\"]}"
+                        "url": "/d/podyamllinks/podyaml?orgId=1&var-uid=${__data.fields[\"PodUID\"]}"
```

Comment:
@linuzb PTAL

Reply:
In Grafana, you don't need a domain and can jump over directly

Reply:
@linuzb currently "/d" supports link redirection

Reply:
SGTM